devices/YarpRobotLoggerDevice/app/robots/iCubGenova09/yarp-robot-logger.xml

```diff
@@ -6,6 +6,7 @@ GNU Lesser General Public License v2.1 or any later version. -->
 <device  xmlns:xi="http://www.w3.org/2001/XInclude" name="yarp-robot-logger" type="YarpRobotLoggerDevice">
   <param name="robot">icub</param>
   <param name="sampling_period_in_s">0.01</param>
+  <param name="rgb_cameras_fps">(20)</param>
```

Comment:
This is problematic. The fps depends according to the camera and the resolution. Different cameras may support different FPS given a resolution. I recently noticed that the video was corrupted in the latest logs because the FPS was not matching the one supported by the camera.